tardis/energy_input/gamma_ray_transport.py

```diff
@@ -578,15 +488,15 @@ def calculate_energy_per_mass(decay_energy, raw_isotope_abundance, cell_masses):
     return energy_per_mass, energy_df
 
 
-def distribute_packets(decay_energy, energy_per_packet):
+def distribute_packets(decay_energy, total_energy, num_packets):
     packets_per_isotope = {}
     for shell, isotopes in decay_energy.items():
         packets_per_isotope[shell] = {}
         for name, isotope in isotopes.items():
             packets_per_isotope[shell][name] = {}
             for line, energy in isotope.items():
                 packets_per_isotope[shell][name][line] = int(
-                    energy / energy_per_packet
+                    energy / total_energy * num_packets
```

Comment:
Is this related to the dataframe, or just a style change to the equation here? `energy_per_packet = total_energy / num_decays` so this is identical, right? `num_packets` is perhaps not the correct name either?

Reply:
Style change. I wanted to see if there is any difference in rounding off. But it is not related to any dataframe.